Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[workspace]
+members = ["ceramic-kubo-rpc", "ceramic-one"]
+
+[workspace.dependencies]
+anyhow = "1"
```

Comment:
If we use full versions here, we can use dependency bot.

Reply:
Hmm can dependency bot use the Cargo.lock file? That would be ideal so we do not get unintentionally stuck on older versions.

Reply:
I usually don't include the Cargo.lock file. You can package it with the release if you want repeatable builds, although having the fully specified versions is usually enough.

Reply:
If we specify each version precisely in Cargo.toml then we loose the ability to use `cargo update` to update to minor/patch versions. The workflow I am aiming for is that we lock our dependencies by default (so that we have reproducible builds) but then we can easily update to new non breaking versions of dependencies explicitly. 

So I would prefer we only specify major versions or minor version in the case of 0. releases. Then periodically we explicitly run `cargo update` to ensure we are getting the most recent versions of our dependencies. 

From [docs](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html):

> An update is allowed if the new version number does not modify the left-most non-zero digit in the major, minor, patch grouping. 

Thoughts?

Reply:
Also from a read of this https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph#supported-package-ecosystems  it looks like dependabot can read the Cargo.lock file. I can verify once we get this initial PR merged.